app/assets/javascripts/config/angularRoutes.js.coffee

```diff
@@ -378,19 +378,6 @@
           AccountService.openAlreadySubmittedModal($stateParams.alreadySubmittedId, $stateParams.doubleSubmit)
       ]
     })
-    .state('dahlia.my-favorites', {
```

Comment:
secret path to my-favorites  with placeholder data that you got if you were logged in.